resources/views/localizations/app.blade.php

```diff
@@ -79,23 +79,30 @@
                     @foreach ($expressions as $key => $value)
                     {{-- Not shown if the expression have an ongoing, unapproved change --}}
                     @php
-                    $duplicate = App\Models\LocalizationContribution::where('language', App::getLocale())->where('key',  $fname.'.'.$key)->where('approved', false)->first();
+                    $translations = App\Models\LocalizationContribution::where('language', App::getLocale())->where('key',  $fname.'.'.$key);
```

Comment:
Possible optimization: move this (and other php parts) to controller and use eager loading.